extras/lsu/configuration_files/HPL.ini

```diff
@@ -0,0 +1,85 @@
+; Path to the csv file that contains the CONTENTdm to MODS mappings.
+mapping_csv_path = 'extras/lsu/mappings_files/HPL.csv'
+; Include the migrated from uri into your generated metadata
+include_migrated_from_uri = true
+; repeatable_wrapper_elements[] = any element that appears on different depths in the xml.
+; Unless specified, MIK will merge fields with the same name even though they belong on separate levels.
+repeatable_wrapper_elements[] = location
+repeatable_wrapper_elements[] = titleInfo
+repeatable_wrapper_elements[] = name
+repeatable_wrapper_elements[] = namePart
+repeatable_wrapper_elements[] = roleTerm
+repeatable_wrapper_elements[] = role
+repeatable_wrapper_elements[] = subject
+repeatable_wrapper_elements[] = topic
+repeatable_wrapper_elements[] = dateCreated
+repeatable_wrapper_elements[] = dateIssued
+repeatable_wrapper_elements[] = publisher
+; repeatable_wrapper_elements[] = physicalDescription
+; repeatable_wrapper_elements[] = extension
+; repeatable_wrapper_elements[] = originInfo
+
+[FILE_GETTER]
+class = CdmSingleFile
+;input_directories[] = "raw_cdmdata/"
+alias = HPL
+ws_url = "https://server16313.contentdm.oclc.org/dmwebservices/index.php?q="
+utils_url = "https://cdm16313.contentdm.oclc.org/utils/"
+
+[WRITER]
+class = CdmSingleFile
+alias = HPL
+output_directory = "output/HPL"
+; specify the datastream desired: MODS or OBJ, or comment out completely for all.
+; datastreams[] = MODS
+; postwritehooks[] = "/usr/bin/php extras/scripts/postwritehooks/sample.php"
+; postwritehooks[] = "/usr/bin/python extras/scripts/postwritehooks/sample.py"
+postwritehooks[] = "/usr/bin/php extras/scripts/postwritehooks/apply_xslt.php"
+postwritehooks[] = "/usr/bin/php extras/scripts/postwritehooks/validate_mods.php"
+
+[MANIPULATORS]
+; One or more fetchermanipulators classes
+; fetchermanipulators[] = "CdmSingleFileByExtension|jp2"
+; fetchermanipulators[] = "RandomSet|50"
+; fetchermanipulators[] = "CdmNoParent"
+; One or more filemanipulators classes.
+; filemanipulators[] = filemanipulator0
+; filemanipulators[] = filemanipulator1
+; One or more metadatamanipulators classes with params
+; metadatamanipulators[] = "Metadatamanipulator0|param_0|...|param_n"
+metadatamanipulators[] = "AddContentdmData"
+metadatamanipulators[] = "FilterModsnamePart|name"
+metadatamanipulators[] = "FilterModsTopic|subject"
+; NormalizeDate requires two arguments, i'm guessing: 1) cdm date fieldname, 2) mods date fieldname
+metadatamanipulators[] = "NormalizeDate|date|dateIssued"
+
+[LOGGING]
+; full path to log file for mik log files
+path_to_log = "/tmp/mik/output.log"
+path_to_manipulator_log = "/tmp/mik/manipulator.log"
+
+[XSLT]
+stylesheets[] = 'extras/lsu/xsl/noteSystemDetailsdelete.xsl'
+stylesheets[] = 'extras/lsu/xsl/titleNonSort.xsl'
+sytlesheets[] = 'extras/lsu/xsl/blankNameSort.xsl'
+stylesheets[] = 'extras/lsu/xsl/subjectSplit.xsl'
+stylesheets[] = 'extras/lsu/xsl/OrderedTemplates.xsl'
```

Comment:
+1 to adding post-write hook specific sections to the .ini file. We have an outstanding [issue](https://github.com/MarcusBarnes/mik/issues/125) for this.